emop-maven-plugin/src/main/java/edu/cornell/AffectedSpecsMojo.java

```diff
@@ -34,11 +49,92 @@ public void execute() throws MojoExecutionException {
         MessageHandler mh = new MessageHandler();
         compiler.run(arguments, mh);
         IMessage[] ms = mh.getMessages(IMessage.WEAVEINFO, false);
+        computeMapFromMessage(ms);
+        writeMapToFile("txt");
         Writer.writeToFile(Arrays.asList(ms), getArtifactsDir() + File.separator + "join-points");
```

Comment:
If you are already writing your message map to disc, you do not need to also write this "join-points" file